src/View/Helper/HtmlHelper.php

```diff
@@ -659,12 +663,12 @@ public function style(array $data, bool $oneLine = true): string
      * - `fullBase` If true the src attribute will get a full address for the image file.
      * - `plugin` False value will prevent parsing path as a plugin
      *
-     * @param string|array $path Path to the image file, relative to the app/webroot/img/ directory.
+     * @param string $path Path to the image file, relative to the app/webroot/img/ directory.
```

Comment:
```suggestion
     * @param string $path Path to the image file, relative to the webroot/img/ directory.
```

Reply:
We often also use ROOT as string:

    ROOT/webroot/img/

but either way works.

Reply:
Anything non-`app/` is OK.

`app/` is from CakePHP 2.